pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -125,12 +125,14 @@ func pickURN(t *testing.T, urns []resource.URN, names []string, target string) r
 
 func TestMain(m *testing.M) {
 	grpcDefault := flag.Bool("grpc-plugins", false, "enable or disable gRPC providers by default")
+	parallelStepGen := flag.Bool("parallel-step-gen", false, "enable or disable parallel step generation")
```

Comment:
I don't see this flag used anywhere, should we use this in CI as well?

Reply:
I added this so that I could run an arbitrary test or suite with this enabled or disabled. I would expect it to be used in CI when this is in